src/components/interaction-designer/block-types/Core_SetGroupMembershipBlock.vue

```diff
@@ -5,24 +5,28 @@
     </h3>
 
     <fieldset :disabled="!isEditable">
-      <block-name-editor :block="block"/>
-      <block-label-editor :block="block"/>
-      <block-semantic-label-editor :block="block"/>
-
-      <div class="form-group">
-        <label>{{'flow-builder.action-label' | trans}}</label>
-        <vue-multiselect v-model="selectedAction"
-                         track-by="id"
-                         label="name"
-                         :placeholder="'flow-builder.action-placeholder' | trans"
-                         :options="actionsList"
-                         :allow-empty="true"
-                         :show-labels="false"
-                         :searchable="false">
-        </vue-multiselect>
-      </div>
-
-      <group-selector :block="block"/>
+      <block-name-editor :block="block" />
+      <block-label-editor :block="block" />
+      <block-semantic-label-editor :block="block" />
+
+      <!-- TODO: Verify the type of is_member(boolean or string) and then enable validation for this field -->
+<!--      <validation-message :message-key="`block/${block.uuid}/.config.is_member`" #input-control="{ isValid: isGroupAMemberValid }">-->
```

Comment:
Commented this line for now until we get clarity on this field. 
@bzabos Can you confirm if we want is_member to be a boolean?
Based on the [flow spec](https://floip.gitbook.io/flow-specification/layers/blocks#set-group-membership) it can be either boolean or an expression 
cc: @safydy @safydy-r

Reply:
Nice one! This one should be an expression editor; if the author would like to hard-code a boolean, I believe they can do so with a static expression! :)